frontend/src/features/properties/filter/PropertyFilter.test.tsx

```diff
@@ -182,12 +183,10 @@ describe('MapFilterBar', () => {
       classificationId: '0',
       minLotSize: '10',
       maxLotSize: '20',
-      inSurplusPropertyProgram: 'true',
+      inSurplusPropertyProgram: TrueFalse.True,
```

Comment:
Ha see, they're so much better than magic strings.

Reply:
Haha the magic strings will slowly disappear...